src/bin/latency.rs

```diff
@@ -361,7 +444,12 @@ fn enable_rx_timestamp(sock: &tsn::TsnSocket, iov: &mut libc::iovec) -> Result<m
         Ok(msg)
     }
 }
-fn get_timestamp(msg: msghdr) -> Result<SystemTime, String> {
+
+fn get_tx_timestamp(fd: i32) -> SystemTime {
+    SystemTime::now()
```

Comment:
This is not TX timestamp.
Even if we currently doesn't support HW tx timestamp, Why not mention it